### PR TITLE
Added an air tanker strike mechanic to Fire Watch mission (and add difficulty modes)

### PR DESCRIPTION
**Pull request by OcelotWalrus**
This PR adds a support power to the player, allowing him to send airstrikes (that drops water) onto highly or unreachable incendiary zones. It takes 50 seconds to reload (on normal speed), and is quite efficient. The air tanker aircraft got an unique sprite.

It's really useful too because some parts of the forest are not reachable by tanks (there are a few starting fires that ain't reachable). I tested the mission with these changes and it was balanced in my opinion.

EDIT:
The PR now adds difficulty modes to the mission:

**EASIER**:
Screenshot from 2024-08-09 12-37-24
**NORMAL**:
Screenshot from 2024-08-09 12-37-40
**HARDER**:
Screenshot from 2024-08-09 12-37-57

### CODE REVIEW

**Comment by Dzierzan** [comment]
Alright, gonna review here mainly the art itself.

IMO it's not done correctly:

- Daniel Cook's artwork has baked in lighting where the light comes from north-west direction. Your artwork almost doesn't include any such lighting, it's too bland.
- Looks like the "nose" doesn't appear to be consistent for each facing. Expecially west and east look odd.
- These sprites look like sprites directly extracted from Magica Voxel without any overpainting where the base model didn't have proper paintjob to begin with.

image

I recommend using either `ship1` or `ship2` since they're right now unused.

ship1

ship2

And before you jump on me, yes, neither I am perfect and I did some mistakes in the past which I am going to slowly start fixing. 

Below you can see an example of redoing sprites done by me:

gunship

**Comment by OcelotWalrus** [comment]
I fixed the spritesheet @Dzierzan, at least I tried to fix it.

**Comment by Dzierzan** [comment]
I dunno, looks a bit better, I give you that. But not good enough. Those sprites have dark tone in general, too dark. Lighting could be even better, making some edge parts more isolated. I also noticed the tail being inconsisent for north and south facings. The same tail look kinda funky for diagonal facings, needs more work too if you want me to even pass that.

Or use `ship1` or `ship2` and call it a day.

**Comment by OcelotWalrus** [comment]
> I dunno, looks a bit better, I give you that. But not good enough. Those sprites have dark tone in general, too dark. Lighting could be even better, making some edge parts more isolated. I also noticed the tail being inconsisent for north and south facings. The same tail look kinda funky for diagonal facings, needs more work too if you want me to even pass that.
> 
> Or use `ship1` or `ship2` and call it a day.

Will work on improving it.

**Comment by Mailaender** [comment]
You can just reuse the existing bomber sprites in the meantime.

**Comment by OcelotWalrus** [comment]
> You can just reuse the existing bomber sprites in the meantime.

Don't worry. I just gotta find some time to do it.

File: mods/hv/maps/firewatch/sequences.yaml [hunk 1]
```diff
@@ -9,14 +9,14 @@ waterball:
 
 watertank:
 	idle:
-		Filename: bits/cvit.png
+		Filename: bits/ship2.png
 		Facings: 8
 	shoot:
-		Filename: bits/tank14.png
+		Filename: bits/ship2.png
```
Comment (Mailaender) [inline mods/hv/maps/firewatch/sequences.yaml:15]: Accidental change?

File: mods/hv/maps/firewatch/sequences.yaml [hunk 2]
```diff
@@ -9,14 +9,14 @@ waterball:
 
 watertank:
 	idle:
-		Filename: bits/cvit.png
+		Filename: bits/ship2.png
```
Comment (Mailaender) [inline mods/hv/maps/firewatch/sequences.yaml:12]: This is an aircraft sprite.

File: mods/hv/maps/firewatch/sequences.yaml (hunk 1 above)
Comment (OcelotWalrus) [reply mods/hv/maps/firewatch/sequences.yaml:15]: No it ain't. I don't find it to be an aircraft, it looked just fine.

Comment (Mailaender) [reply mods/hv/maps/firewatch/sequences.yaml:15]: I wanted something like this https://www.polizeimuseum-stuttgart.de/oldtimer-das-rollende-museum/unser-fuhrpark/sonderfahrzeuge/ so please revert.

Comment (OcelotWalrus) [reply mods/hv/maps/firewatch/sequences.yaml:15]: Sure, no problem.

**Comment by OcelotWalrus** [comment]
The air tanker now uses the bomber sprite, but I still let my airtanker sprite in the mission's folder, to implement it in the future, once it's enhanced.

**Comment by Mailaender** [comment]
Put it on a separate branch in your repository, please.

File: mods/hv/maps/firewatch/sequences.yaml [hunk 3]
```diff
@@ -40,3 +40,9 @@ firefighterpod:
 		Offset: 0,-2
 	icon:
 		Filename: maps/firewatch/firefighterpod-icon.png
+
+airtanker:
+	idle:
+		Filename: maps/firewatch/bomber.png
```
Comment (Mailaender) [inline mods/hv/maps/firewatch/sequences.yaml:46]: > maps/firewatch/bomber.png not found

**Comment by Mailaender** [comment]
Needs a rebase.

**Pull request merged**